client/src/service/handlers.js

```diff
@@ -0,0 +1,14 @@
+export const pullOutJson = (response) => {
+    const contentType = response.headers.get('content-type');
+    if (contentType && contentType.includes('application/json')) {
+        return response.json();
+    }
+    throw new TypeError('Oops, we haven\'t got JSON!');
```

Comment:
I don't think we need to account for not getting back results in json format because IIRC according to the docs, they always return json format no?

We also don't catch errors in the fetch anyways so not sure if this is needed or not but definitely a good practice 💯💯

Reply:
Based on their documentation the format is json, but it doesn't hurt to have this I think. (In case we're making calls to incorrect APIs later or something, it would be easier to identify.)